packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -70,13 +70,18 @@ dependencies {
     implementation "com.github.wordpress-mobile:react-native-slider:${extractPackageVersion(packageJson, '@react-native-community/slider', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-get-random-values:${extractPackageVersion(packageJson, 'react-native-get-random-values', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-masked-view:${extractPackageVersion(packageJson, '@react-native-masked-view/masked-view', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-gesture-handler:${extractPackageVersion(packageJson, 'react-native-gesture-handler', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-screens:${extractPackageVersion(packageJson, 'react-native-screens', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-safe-area-context:${extractPackageVersion(packageJson, 'react-native-safe-area-context', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-reanimated:${extractPackageVersion(packageJson, 'react-native-reanimated', 'dependencies')}"
+    implementation 'com.github.wordpress-mobile:react-native-reanimated:9e63336aa24daeefb9f756bbf8cdddfa2f3d44f5'
     implementation "com.github.wordpress-mobile:react-native-prompt-android:${extractPackageVersion(packageJson, 'react-native-prompt-android', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-webview:${extractPackageVersion(packageJson, 'react-native-webview', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-clipboard:${extractPackageVersion(packageJson, '@react-native-clipboard/clipboard', 'dependencies')}"
+
+    implementation("com.github.wordpress-mobile:react-native-gesture-handler:${extractPackageVersion(packageJson, 'react-native-gesture-handler', 'dependencies')}", {
+        // Remove Reanimated transitive dependency as it's already defined here
+        exclude group: 'com.github.wordpress-mobile', module: 'react-native-reanimated'
+    })
```

Comment:
The transitive dependency is defined [here](https://github.com/wordpress-mobile/react-native-gesture-handler/blob/1a4713873e9aeba0616f3f1e180595e6101cd3bd/android/build.gradle#L94). Once https://github.com/wordpress-mobile/react-native-reanimated/pull/15 is merged, we could update it so the same version is referenced in both places. I'd like to note that this is not mandatory as it won't affect the editor, however, it's recommended to prevent build issues in `react-native-gesture-handler` due to using different versions.